assets/js/src/site/plugins/pum-analytics.js

```diff
@@ -18,12 +18,12 @@ var PUM_Analytics;
             var beacon = new Image(),
                 url = rest_enabled ? pum_vars.restapi : pum_vars.ajaxurl,
                 opts = {
-                    route: '/analytics/',
-                    data: $.extend({
```

Comment:
@danieliser Will any of these changes within analytics affect anyone with a current version of Popup Analytics?

Reply:
@fpcorso No, the current version uses an actual AJAX call to admin-ajax.php (terrible idea right lol). Theirs will still work as normal.

Also the highlighted lines didn't actually change, just got a filter added to them so they can be extended in the new version.